docs/installation.rst

```diff
@@ -51,6 +51,43 @@ Now use pip to install packages (this will only install DESC + JAX with CPU capa
     # optionally install developer requirements (if you want to run tests)
     pip install -r devtools/dev-requirements.txt
 
+**Or using uv instead of pip**
+
+One could use `uv <https://docs.astral.sh/uv>`_, a new python package management tool, instead of pip.
+For a project that modifies DESC and also uses it to perform analysis,
+it can be nice to separate the DESC folder from the project's data, scripts, jupyter notebooks, etc.
+This will show how to set up a new ``uv`` project with DESC as an editable dependency,
```

Comment:
```suggestion
This will show how to set up a new ``uv`` project with DESC as an editable dependency (Either on local machine or on the cluster, this method can work with both),
```